pkgs/os-specific/linux/kernel/manual-config.nix

```
@@ -368,7 +368,7 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.linux-kernel kernelPat
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [ perl bc nettools openssl rsync gmp libmpc mpfr zstd python3Minimal ]
-      ++ optional  (stdenv.hostPlatform.linux-kernel.target == "uImage") buildPackages.ubootTools
+      ++ optional  (stdenv.hostPlatform ? linux-kernel && stdenv.hostPlatform.linux-kernel.target == "uImage") buildPackages.ubootTools
```

I think this might help with OfBorg

```suggestion
      ++ optional  (stdenv.hostPlatform.linux-kernel.target or null == "uImage") buildPackages.ubootTools
```

---

Let's try it.